jwt/src/test/java/me/uport/sdk/jwt/JWTToolsJVMTest.kt

```diff
@@ -2,36 +2,38 @@ package me.uport.sdk.jwt
 
 import com.uport.sdk.signer.KPSigner
 import kotlinx.coroutines.runBlocking
-import me.uport.sdk.core.stubUiContext
-import org.junit.Assert.assertNull
+import me.uport.sdk.core.Networks
+import me.uport.sdk.ethrdid.EthrDIDResolver
+import me.uport.sdk.jsonrpc.JsonRPC
+import me.uport.sdk.universaldid.UniversalDID
+import me.uport.sdk.uportdid.UportDIDResolver
+import org.junit.Assert.assertNotNull
 import org.junit.Before
 import org.junit.Test
-import java.util.concurrent.CountDownLatch
-import java.util.concurrent.TimeUnit
 
 class JWTToolsJVMTest {
 
     @Before
     fun `run before every test`() {
-        stubUiContext()
+//        stubUiContext()
```

Comment:
since JWTTools() constructor should already be handling this, there is no need for this setup.